src/common/locales/strings/enUS.json

```diff
@@ -36,7 +36,7 @@
   "change_recovery_question4": "What was your favorite brand of clothing in high school?",
   "change_recovery_question5": "What is the birthdate of your favorite grand parent?",
   "change_recovery_question6": "What is the birthdate of your favorite niece or nephew?",
-  "change_recovery_question8": "What was your address in college?",
+  "change_recovery_question8": "What was your street address in college?",
```

Comment:
This wasn't part of the requested changes. 

Also, the other languages remain unchanged. To enact a string change, we need to change the key so it gets re-translated.

Reply:
Also in the TOS string modification, only English was changed.